### PR TITLE
[optimizer] Momentum

### PR DESCRIPTION
**Pull request by RodionovF**

### CODE REVIEW

File: include/deepworks/optimizer.hpp [hunk 1]
```diff
@@ -21,5 +21,22 @@ class SGD {
     Parameters& m_params;
 };
 
+class Momentum {
```
Comment (TolyaTalamanov) [inline include/deepworks/optimizer.hpp:24]: Does it work better for your CIFAR network ?

**Review by TolyaTalamanov — APPROVED** [review]

**Comment by TolyaTalamanov** [comment]
@l-bat Have a look, please

File: include/deepworks/optimizer.hpp (hunk 1 above)
Comment (l-bat) [reply include/deepworks/optimizer.hpp:24]: Maybe rename it to `SGDMomemtum`? Because momentum is used in other optimizers too

**Review by l-bat — APPROVED** [review]

File: include/deepworks/optimizer.hpp [hunk 2]
```diff
@@ -21,5 +21,22 @@ class SGD {
...
+public:
+    explicit Momentum(Parameters& params, float lr, float gamma);
```
Comment (l-bat) [inline include/deepworks/optimizer.hpp:26]: I think we can set `gamma = 0.9f` by default

**Comment by TolyaTalamanov** [comment]
Cooool :+1: :fire:

**Pull request merged**